packages/gatsby-source-drupal/src/utils.js

```diff
@@ -154,7 +154,8 @@ const handleWebhookUpdate = async (
   }
 
   // download file
-  if (isFileNode(newNode)) {
+  const { skipFileDownloads } = pluginOptions
+  if (isFileNode(newNode) && skipFileDownloads !== false) {
```

Comment:
This looks suspicious. "If file node and skipFileDownloads is true, then download the file"?

Also, above it is is set to `false` if it falsy. This case is a strict comparison. Seems to me like both usages should do the same thing.